### PR TITLE
Revive lost test...

### PR DESCRIPTION
**Pull request by foolswood**
as for whether we want it, dunno.

### CODE REVIEW

File: types/test/test_chronology.c [hunk 1]
```diff
@@ -8,16 +8,18 @@ typedef struct {
...
+// or should go.
+static rage_Error error_context(rage_Error (*f)(StringBuffer b)) {
     const unsigned buf_size = 80;
     char buf[buf_size];
     StringBuffer sb = {.buf=buf, .size=buf_size};
     rage_Error e = f(sb);
     if (RAGE_FAILED(e)) {
         printf("%s\n", RAGE_FAILURE_VALUE(e));
-        return 1;
+	return RAGE_ERROR("Failed in error context");
```
Comment (ch3pjw) [inline types/test/test_chronology.c]: This indentation looks squiffy...

File: types/test/test_chronology.c [hunk 2]
```diff
@@ -8,16 +8,18 @@ typedef struct {
     char * buf;
 } StringBuffer;
 
-int error_context(rage_Error (*f)(StringBuffer b)) {
+// FIXME: This really either ought to go all the way to the top of the testing
```
Comment (ch3pjw) [inline types/test/test_chronology.c]: Does "all the way to the top of the testing" mean "test the whole stack" or "run first"?

File: types/test/test_chronology.c [hunk 3]
```diff
@@ -32,7 +34,8 @@ int error_context(rage_Error (*f)(StringBuffer b)) {
         return RAGE_ERROR(eb.buf); \
     }
 
-rage_Error test_time_delta(StringBuffer eb) {
+// FIXME: this seems to be the only thing exercising this code, remove?
```
Comment (ch3pjw) [inline types/test/test_chronology.c]: As in, the code is not used in production, just by this test?

File: types/test/test_main.c [hunk 4]
```diff
@@ -16,4 +16,5 @@ TEST_MAIN(
     interpolator_immediate_change_test,
     test_ports,
     test_time_series_new,
-    test_time_after)
+    test_time_after,
+    test_time_delta)
```
Comment (ch3pjw) [inline types/test/test_main.c:20]: Were these tests just "lost" in the sense that they'd dropped out of this list?

**Review by ch3pjw — COMMENTED** [review]
Just a few questions - I feel like I'm missing a little context on the FIXMEs.

File: types/test/test_chronology.c (hunk 3 above)
Comment (foolswood) [reply types/test/test_chronology.c]: Yes, I wrote all the time helpers quite early and ended up not really using many of them at all.

File: types/test/test_chronology.c (hunk 2 above)
Comment (foolswood) [reply types/test/test_chronology.c]: Neither, I mean "should be an integral part of the test runner", will update.

File: types/test/test_main.c (hunk 4 above)
Comment (foolswood) [reply types/test/test_main.c:20]: Yes, and because they weren't declared static nothing complained.

**Pull request merged**